packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -7230,9 +7235,7 @@ describe('plugin-meetings', () => {
             assert.equal(meeting.hasMediaConnectionConnectedAtLeastOnce, false);
 
             // simulate first connection success
-            eventListeners[Event.CONNECTION_STATE_CHANGED]({
-              state: 'Connected',
-            });
+            eventListeners[Event.PEER_CONNECTION_STATE_CHANGED]();
```

Comment:
any reason why we aren't using the simulate function here?

Reply:
Thought that we are setting 'connected' state at the begging, so it's enough to trigger event here, but you are right that using simulate function makes the test more clear. I will update test